lib/src/features/authentication/view/signin_view.dart

```diff
@@ -67,48 +76,42 @@ class SignInView extends StatelessWidget {
   }
 
   Widget _buildForm(BuildContext context) {
-    return Column(
-      children: [
-        XInput(
-          key: Key(S.of(context).sign_in_key_username),
-          value: '',
-          /*
-          Will be developed in logic task
-          onChanged: context.read<SigninBloc>().onEmailChanged,
-          */
-          onChanged: null,
-          style: AppStyles.whiteTextMidium,
-          labelStyle: AppStyles.whiteTextMidium,
-          keyboardType: TextInputType.emailAddress,
-          decoration: InputDecoration(
-            labelText: S.of(context).sign_in_accout_lable,
-            /*
-            Will be developed in logic task
-            errorText: 'state.email.errorOf(context)',
-            */
-          ),
-        ),
-        const SizedBox(height: 16.0),
-        XInput(
-          key: Key(S.of(context).sign_in_key_password),
-          value: 'password',
-          /*
-          Will be developed in logic task
-          onChanged: context.read<SigninBloc>().onPasswordChanged,
-          */
-          onChanged: null,
-          obscureText: true,
-          style: AppStyles.whiteTextMidium,
-          labelStyle: AppStyles.whiteTextMidium,
-          decoration: InputDecoration(
-            labelText: S.of(context).sign_in_password_lable,
-            /*
-            Will be developed in logic task
-            errorText: 'state.password.errorOf(context)',
-            */
-          ),
-        ),
-      ],
+    return BlocBuilder<SigninBloc, SigninState>(
+      buildWhen: (previous, current) {
+        return previous.email != current.email ||
+            previous.password != current.password;
+      },
+      builder: (context, state) {
+        return Column(
+          children: [
+            XInput(
+              key: Key(S.of(context).sign_in_key_username),
+              value: state.email.value,
+              onChanged: context.read<SigninBloc>().onEmailChanged,
+              style: AppStyles.whiteTextMidium,
+              labelStyle: AppStyles.whiteTextMidium,
+              keyboardType: TextInputType.emailAddress,
+              decoration: InputDecoration(
+                labelText: S.of(context).sign_in_accout_lable,
+                errorText: state.email.errorOf(context),
+              ),
+            ),
+            const SizedBox(height: 16.0),
+            XInput(
+              key: Key(S.of(context).sign_in_key_password),
+              value: state.password.value,
+              onChanged: context.read<SigninBloc>().onPasswordChanged,
+              obscureText: true,
+              style: AppStyles.whiteTextMidium,
+              labelStyle: AppStyles.whiteTextMidium,
+              decoration: InputDecoration(
+                labelText: S.of(context).sign_in_password_lable,
+                errorText: state.password.errorOf(context),
```

Comment:
Please separate the code to `_inputEmail`, `_inputPassword` and use `buildWhen` only `email` or `password`